src/mission-planner/nodes/server.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
```

Comment:
So this server is the thing that will be listening to the dynamic_reconfigure client and making changes as needed. For now, the callback function is just logging stuff.

Reply:
Just for plannings sake, how would the mission planner react to receiving new configuration? Should it just restart everything with a new mission plan? Would it have a subset of parameters that control the mission and require restarting missions?

Reply:
I think restarting everything with a new mission plan is the best way at the moment. I will try looking around to see if there are any other examples of ways people use dynamic reconfigure that we may be able to pull from.